File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/controller/Main.java
package com.sergey.pisarev.controller;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.util.Objects;

public class Main extends Application {

    public static String title = "CNC modeling 2.0";

    @Override
    public void start(Stage primaryStage) throws Exception {
        MainController.STAGE = primaryStage;
        Parent root = FXMLLoader.load(Objects.requireNonNull(getClass().getClassLoader().getResource("main.fxml")));
        primaryStage.setTitle(title);
        primaryStage.setScene(new Scene(root, 1000, 600));
        primaryStage.setMinHeight(600);
        primaryStage.setMinWidth(1000);
        primaryStage.show();
    }

    public static void main(String[] args) {
        launch(args);
    }
}

File: CNC modeling 2.0/src/main/java/com/sergey/pisarev/controller/Main.kt
package com.sergey.pisarev.controller

import kotlin.Throws
import java.lang.Exception
import com.sergey.pisarev.controller.MainController
import javafx.application.Application
import javafx.fxml.FXMLLoader
import javafx.scene.Parent
import javafx.scene.Scene
import javafx.stage.Stage
import java.util.Objects
import kotlin.jvm.JvmStatic

class Main : Application() {
    @Throws(Exception::class)
    override fun start(primaryStage: Stage) {
        MainController.STAGE = primaryStage
        val root = FXMLLoader.load<Parent>(Objects.requireNonNull(javaClass.classLoader.getResource("main.fxml")))
        primaryStage.title = title
        primaryStage.scene = Scene(root, 1000.0, 600.0)
        primaryStage.minHeight = 600.0
        primaryStage.minWidth = 1000.0
        primaryStage.show()
    }

    companion object {
        var title = "CNC modeling 2.0"
        @JvmStatic
        fun main(args: Array<String>) {
            launch(Main::class.java)
        }
    }
}